tests/identify_test.py

```diff
@@ -179,12 +179,14 @@ def test_tags_from_interpreter(interpreter, expected):
         (b'', True),
         ('éóñəå  ⊂(◉‿◉)つ(ノ≥∇≤)ノ'.encode('utf8'), True),
         (r'¯\_(ツ)_/¯'.encode('utf8'), True),
-        ('♪┏(・o･)┛♪┗ ( ･o･) ┓♪┏ ( ) ┛♪┗ (･o･ ) ┓♪┏(･o･)┛♪'.encode('utf8'), True),
```

Comment:
This is emoji murder in the name of style checking satisfactimacation. You should be ashamed.